lesson15/app.py

```diff
@@ -0,0 +1,20 @@
+from car import Car
+from home import Home
+from human import Human
+
+
+def app():
+    alex = Human('Alex')
+    car = Car('Porsche', '911')
+    home = Home('blvd. Riverside', '2505')
+    for _ in range(40):
+        alex.work()
+    alex.buy(car)
```

Comment:
You pass instance of the `Car` here.. but there you accept entry some `person`.. strange..